bin/asset-size-tracking/src/post-comment.sh

```diff
@@ -33,9 +33,10 @@ API_VERSION=v3
 API_HEADER="Accept: application/vnd.github.${API_VERSION}+json; application/vnd.github.antiope-preview+json"
 AUTH_HEADER="Authorization: token ${GITHUB_TOKEN}"
 
-delete_comment_if_exists() {
+update_comment_if_exists() {
```

Comment:
Unrelated to this PR but as these grow/become more complicated should we rewrite them in js?